src/main/java/org/recordrobotics/charger/subsystems/Claw.java

```diff
@@ -0,0 +1,36 @@
+package org.recordrobotics.charger.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import org.recordrobotics.charger.RobotMap;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class Claw extends SubsystemBase {
+	private static final double GEAR_RATIO = 63;
+
+	public static final double CLAW_NEUTRAL = 0.0;
+	public static final double CLAW_CUBE = 0.2;
+	public static final double CLAW_CONE = 0.3;
```

Comment:
these variables seem vague to me. at the very least, add a comment above them stating what they represent

Reply:
these are placeholders

Reply:
placeholders for what?

Reply:
the comments describe what the terms are meant to represent. they should more be what the numbers are for, like `encoder value of position`, which is what I assume these are

Reply:
I still don't think this is done yet. yes, neutral, cube, and cone are claw positions, but what do the numbers tied to those variables represent?

Reply:
In our code "position" and "encoder value" are used interchangeably, so I don't think this is a problem.